sources/api/apiserver/src/server/mod.rs

```diff
@@ -89,7 +130,8 @@ where
             .service(
                 web::scope("/settings")
                     .route("", web::get().to(get_settings))
-                    .route("", web::patch().to(patch_settings)),
+                    .route("", web::patch().to(patch_settings))
+                    .route("/keypair", web::patch().to(patch_settings_key_pair)),
```

Comment:
Yes that is true. 

I used keypair as a path after checking the current GET for specific settings is part of the query: `/settings?keys=foo,bar,baz`  .  I preferred to just add a Path "/keypair" to "/settings" since it falls under the settings umbrella while still being specific enough only for "keypair" type inputs. 

I do like the approach of using a top level /settings-keypair. But since we wouldnt really support GET's from this endpoint, I preferred to keep the more specific `keypair` under the `/settings` umbrella.

Reply:
Are we assuming then that an OOTB user would never create a setting named `keypair`? i.e. if I made `settings.keypair` a part of my variant's schema, this would break?

Reply:
If a user creates a settings with the name `keypair`, it would work as is. For example, 

```
apiclient set keypair=test # works
```
because it would call the /settings/keypair endpoint but with this payload

```
{"request_payload": "{\"keypair\": \"test\"}"}
```

So this should work as is. Ben was hinting at the fact that if we ever introduce a /get with a path based structure like GET /settings/{SETTINGS_NAME} like /settings/motd or in this case, /settings/keypair , it would lead to confusion. (It would still work, since the type is PATCH)